docs/running-a-node/ledger-management.md

```diff
@@ -28,24 +30,29 @@ Depending on machine and networking resources, the bootstrap performance can be
 
 Before using this method there are a few considerations to ensure it is done safely:
 
 ### Data source
 Make sure you trust the source providing the data to you. If you are unfamiliar with the individual or organization providing the ledger, consider other options for the data or fallback to the default of [bootstrapping](#bootstrapping) from the network.
 
 ### Representative voting weights
-Blocks are confirmed using the voting weight of representatives and these weights are determined by the account balances assigned to those representatives. In addition, the node releases contain a hard-coded set of representative weights captured at the time of release.
+Blocks are confirmed using the voting weight of representatives and these weights are determined by the account balances assigned to those representatives. In addition, the node releases contain a hard-coded set of representative weights captured at the time of the node release to help this process during bootstrapping.
+
+If looking to use a downloaded ledger there is a risk of it providing inaccurate representative voting weights. Although the potential impacts of this are minimal, below are some recommended steps to take which can help provide additional confidence the ledger can be used.
 
-If a new ledger is downloaded it is recommended to compare the voting weights it contains against the hard-coded values in the node. To do this, the [--compare_rep_weights](/commands/command-line-interface/#-compare_rep_weights) CLI (_available in V21+ only_) can be run to get an output of the mismatched weight, mean and standard deviation.
+1. **Scan the ledger for integrity using the [`--debug_validate_blocks`](/commands/command-line-interface/#-debug_validate_blocks) CLI command**. If issues are found they should be inspected carefully and alternative sources of a ledger may need to be considered as failures with this command have a high chance of indicating potentially malicious behavior.
+1. With the new ledger in the data folder (don't forget to backup the previous ledger!), **review the differences in representative voting weights by running the [`--compare_rep_weights`](/commands/command-line-interface/#-compare_rep_weights) CLI command**. This will compare the new ledger voting weights against the hardcoded values in the node (set at the time of release). See the CLI command for details on the output, but special attention should be paid to entries in the `outliers` section, which indicates large differences between the hard coded and new ledger rep weights, and the `newcomers` section, which highlights large voting weights showing up for new representatives. By inspecting the output addresses in public explorers such as [Nanocrawler.cc](https://nanocrawler.cc), this can help to determine if voting weight may have been manipulated in the downloaded ledger.
```

Comment:
`--data_path` can be used to point the command to another ledger so that there's no need to backup the current one and move things around. For a ledger in `/path/to/ledger/data.ldb` , the ccommand would be `--compare_rep_weights --data_path /path/to/ledger/`

Reply:
Tweaked this to mentioned `--data_path` and overall simplify things to push more on the CLI description.